spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -1114,21 +1114,116 @@ describe('document-capture/components/acuant-capture', () => {
   });
 
   context('mobile selfie', () => {
-    it('renders the selfie capture loading div in acuant-capture', async () => {
+    const trackEvent = sinon.stub();
+
+    beforeEach(async () => {
+      // Set up the components so that everything is as it would actually be -except- the AcuantSDK
+      // The AcuantSDK isn't possible to run in test, so the initialize({...}) call below mocks it.
+      render(
+        <DeviceContext.Provider value={{ isMobile: true }}>
+          <AnalyticsContext.Provider value={{ trackEvent }}>
+            <AcuantContextProvider sdkSrc="about:blank" cameraSrc="about:blank">
+              <AcuantCapture label="Image" name="selfie" isReady />
+            </AcuantContextProvider>
+          </AnalyticsContext.Provider>
+        </DeviceContext.Provider>,
+      );
+
+      // Simulate the user clicking on the box that usually opens full screen selfie capture.
+      // This isn't strictly necessary for the logging tests, but doing this makes the calls to
+      // trackEvent appear in the actual order we'd expect when using the Acuant SDK.
+      await userEvent.click(screen.getByLabelText('Image'));
+    });
+
+    it('renders the selfie capture loading div in acuant-capture', () => {
       // What we want to test is that the selfie version of the FileInput appears
       // when the name="selfie". The only difference between the selfie and document
       // versions is what happens when you click the FileInput, so this test clicks
       // the file input, then checks that the full screen div opened
-      const { getByRole, getByLabelText } = render(
-        <DeviceContext.Provider value={{ isMobile: true }}>
-          <AcuantContextProvider sdkSrc="about:blank" cameraSrc="about:blank">
-            <AcuantCapture label="Image" name="selfie" isReady />
-          </AcuantContextProvider>
-        </DeviceContext.Provider>,
+      expect(screen.getByRole('dialog')).to.be.ok();
+    });
+
+    it('calls trackEvent from onSelfieCaptureOpen', () => {
+      // In real use the `start` method opens the Acuant SDK full screen selfie capture window.
```

Comment:
I wasn't sure if it would be better to only list this comment once and write something like `see test for "OnSelfieCaptureOpen" above for more info`. But I thought if the code changed, we could easily lose the main comment, while still referencing it in other comments, so I thought in this case it was better to leave it repeated.

Reply:
Yeah, I agree with your concern about the code changing and causing `see test for "OnSelfieCaptureOpen" above for more info` to lose its meaning.  I'm good with the repeat.